uber/config.py

```diff
@@ -363,6 +363,8 @@ def _is_intstr(s):
 
 c.TABLE_PRICES = defaultdict(lambda: _config['table_prices']['default_price'],
                              {int(k): v for k, v in _config['table_prices'].items() if k != 'default_price'})
```

Comment:
The real use-case here is when an event wants to define a set of tables that aren't integers. In Anthrocon's case, there are several specialized tables, including a half-table. You can see where I define them at: https://github.com/Anthrocon-Reg/anthrocon_plugin/blob/master/anthrocon_plugin/config.py#L6